3d_classification/ignite/densenet_evaluation_array.py

```diff
@@ -30,18 +30,21 @@ def main():
     logging.basicConfig(stream=sys.stdout, level=logging.INFO)
 
     # IXI dataset as a demo, downloadable from https://brain-development.org/ixi-dataset/
+    # the path of ixi IXI-T1 dataset
+    data_path = "/workspace/data/medical/ixi/IXI-T1"
     images = [
-        os.sep.join(["workspace", "data", "medical", "ixi", "IXI-T1", "IXI607-Guys-1097-T1.nii.gz"]),
```

Comment:
please don't change the path, your raw path can't support `windows` OS.

Reply:
I see, my purpose is to use a variable to represent the root path, thus user is easier to modify it. Let me use`os.path.join` to replace the `/`